doc/config.rst

```diff
@@ -1940,6 +1940,13 @@ These options influence text output.
 
    .. versionadded:: 1.1
 
+.. confval:: text_maxwidth
+
+   The maximum width of line.
+
+   The default is ``70``.
+
+   .. versionadded:: 1.6
```

Comment:
Sorry, we called feature-freeze for 1.6 now. So this should be added into 1.7 (master branch).